src/nxdoc/tutorials/transversal-how-tos/how-to-implement-local-groups-or-roles-using-computed-groups.md

```diff
@@ -146,6 +146,21 @@ In this how to, you will learn how to let managers of a workspace determine who
 
 {{/callout}}
 
+Starting from 8.10 HF-04 you need to add the following contribution.
+
+{{#> panel type='code'}}
+
+```xml
+<extension point="configuration" target="org.nuxeo.runtime.ConfigurationService">
+    <documentation>
+      Allow setting permissions to virtual groups
+    </documentation>
+    <property name="nuxeo.automation.allowVirtualUser">true</property>
+  </extension>
```

Comment:
Please delete the leading spaces of each line to have a better indentation (2 spaces).